docs/src/hugo/config.toml

```diff
@@ -1,8 +1,11 @@
-baseurl = "http://http4s.org/"
+baseurl = "http://http4s.org/v0.15"
+contentDir = "../../target/scala-2.12/tut"
 languageCode = "en-us"
 title = "http4s"
+theme = "http4s.org"
+themesDir = "../../../website/src/hugo/themes"
```

Comment:
This keeps the common branding elements in one place.  It won't automatically refresh old versions of `docs`, but it will keep the current version of `docs` looking about like the `website`.

Note that the static assets herein will be copied into the `vX.Y` branch.  This is critical to letting us continue to evolve the theme on `website`.